docs/getting-started.md

```diff
@@ -7,22 +7,29 @@ description: Jump in and get started quickly using Momento Serverless Cache with
 ---
 
 # Getting Started with Momento Serverless Cache
-If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. There are multiple ways to access the service below. 
+If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. Want to quickly get a feel for the Momento experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser). Or, if you're ready to setup your development environment and get to building your own applications with Momento you can follow the [simple instructions below](#install-the-momento-command-line-interface-cli).
 
-## Try our CLI in your browser
+## Try our CLI and an SDK in your browser
 
-You can try out our CLI tool today, right in your browser.  No software installation required!
+You can try out Momento Serverless Cache today, right in your browser. No software installation required! **Note: Momento Serverless Cache is an API based serverless service. It does not deploy anything into your (AWS, GCP, Azure, etc.) account.**
```

Comment:
Bold emphasis seems like a bit much. I might recommend using italics (single asterisk) instead.

Reply:
For the note, can we instead do the docusaruas note style? You can see an example of it here https://docs.momentohq.com/develop/api-reference/collections/dictionary#dictionaryincrement

Reply:
Turned this into note style - looks good Kirk thanks!